source/subs_functions/subs_functions.bas

```diff
@@ -233,11 +233,12 @@ id.hr_syntax = "_MEMIMAGE or _MEMIMAGE(imageHandle)"
 regid
 
 clearid
-id.n = qb64prefix$ + "MemSound": id.Dependency = DEPENDENCY_AUDIO_DECODE
+id.n = qb64prefix$ + "MemSound": id.Dependency = DEPENDENCY_MINIAUDIO
 id.subfunc = 1
 id.callname = "func__memsound"
 id.args = 2
 id.arg = MKL$(LONGTYPE - ISPOINTER) + MKL$(LONGTYPE - ISPOINTER)
+id.specialformat = "?[,?]"

```

Comment:
Huh, was this a bug that the second parameter wasn't optional?

Reply:
Yeah. It wasn't. We should have made it optional the first time. We made it optional with this PR. See the discussion here: https://discord.com/channels/975381912350752819/975389657212805160/1154682098913910844